doc/03_reference/verification.rst

```diff
@@ -223,6 +223,138 @@ You can add any compile/runtime options in `dv/uvm/core_ibex/yaml/simulator.yaml
+| Integration   | PRE_VERIFIED_SUB_MODULES_V2         | Complete   | ICache is verified in a seperate testbench.                                                                                                                          |
++---------------+-------------------------------------+------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Issues        | NO_HIGH_PRIORITY_ISSUES_PENDING     | Complete   |                                                                                                                                                                      |
++---------------+-------------------------------------+------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Issues        | ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED | Complete   |                                                                                                                                                                      |
++---------------+-------------------------------------+------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Review        | DV_DOC_TESTPLAN_REVIEWED            | Complete   |                                                                                                                                                                      |
++---------------+-------------------------------------+------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Review        | V3_CHECKLIST_SCOPED                 | Complete   |                                                                                                                                                                      |
++---------------+-------------------------------------+------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+**PMP Testing Note**: A large number of iterations of ``pmp_full_random_test`` are required to meet coverage goals and timed out tests must be included in the coverage collection.
+This is because of the large cross bins for PMP that aim to explore the full space of possible behaviour.
+The current strategy of random generation is very inefficient at exploring this space.
+It is also complex to write a randomly generated test that can deal with all possible scenarios without hitting a double faulting or time out scenarios (e.g. consider a random configuration that gives you no executable regions).
```

Comment:
This is especially true when enhanced PMP modes like machine mode lockdown and machine mode whitelist policy are enabled.